configure.ac

```diff
@@ -169,7 +169,9 @@ AC_CHECK_FUNCS([pthread_getname_np pthread_setname_np])
 dnl Check for the XKB extension to get reliable keyboard handling
 AC_CHECK_HEADER(X11/XKBlib.h, AC_DEFINE(HAVE_XKB,1))
 AC_CHECK_HEADERS([netipx/ipx.h linux/ipx.h])
-CONFIG_TIME=`date +"%F %T %z"`
+DATE_FMT="%F %T %z"
+SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH:-$(date +%s)}"
+CONFIG_TIME=$(date -u -d "@$SOURCE_DATE_EPOCH" "+$DATE_FMT" 2>/dev/null || date -u -r "$SOURCE_DATE_EPOCH" "+$DATE_FMT" 2>/dev/null || date -u "+$DATE_FMT")
```

Comment:
What does `@$SOURCE_DATE_EPOCH` mean here?
And too many bashisms, please write as
`if test -n "$SOURCE_DATE_EPOCH"`

Reply:
The @ is not shell magic, but a literal `@` (at) sign as documented in the GNU date man page 
```
EXAMPLES
       Convert seconds since the epoch (1970-01-01 UTC) to a date

              $ date --date='@2147483647'```